addons/web/static/src/js/views/control_panel/mobile_control_panel_renderer.js

```diff
@@ -69,6 +70,18 @@ ControlPanelRenderer.include({
         this.$('.o_mobile_search').toggleClass('o_hidden');
         this._renderSearchviewInput();
     },
+    /**
+     * Display mobile quick search on screen
+     */
+    _mobileQuickSearchView() {
+        this.$('.o_cp_searchview').addClass('o_searchview_quick');
+        this.$('.breadcrumb').addClass('o_hidden');
```

Comment:
Needed? Now withBreadcrumbs is false. So I suppose that the breadcrumb is already hidden.